### PR TITLE
Updates the server to 515

### PR DESCRIPTION
**Pull request by thedarkcrusader**
so basically, a lot of it is just an old   thing that sucked and some of it is switch stuff so we got rid of all of that might be redundant does cause warnings, but it works needs more testing, but it all works the way it should from what i can see is there a better way to do this maybe but better is not my middle name.

### CODE REVIEW

**Comment by thedarkcrusader** [comment]
UH OH  A moment

**Comment by thedarkcrusader** [comment]
i gottta fix the life proc but a moment

**Comment by thedarkcrusader** [comment]
we are good on this now

**Comment by thedarkcrusader** [comment]
fubar

**Comment by thedarkcrusader** [comment]
i will re do this PR

**Comment by mattmatterson111** [comment]
Is this good to merge?

**Comment by thedarkcrusader** [comment]
> Is this good to merge?

Yeah it's all good to go  should be.

**Comment by mattmatterson111** [comment]
Alright. Gonna merge as soon as I get home.

**Comment by mattmatterson111** [comment]
Here we go.

File: code/game/objects/structures/bedsheet_bin.dm [hunk 1]
```diff
@@ -119,7 +119,7 @@ LINEN BINS
 
 
 /obj/structure/bedsheetbin/update_icon()
-	switch(amount)
+	if(amount)
```
Comment (mattmatterson111) [inline code/game/objects/structures/bedsheet_bin.dm:122]: ?????

File: code/modules/mob/living/carbon/carbon.dm [hunk 2]
```diff
@@ -116,7 +116,7 @@
 			Stun(2)
 		if(21 to 25)
 			Weaken(2)
-		if(26 to 30)
+		if(25 to 26)
```
Comment (mattmatterson111) [inline code/modules/mob/living/carbon/carbon.dm:119]: ????

File: code/controllers/subsystems/jobs.dm [hunk 3]
```diff
@@ -476,8 +476,8 @@ SUBSYSTEM_DEF(jobs)
 			G.prescription = 7
 	if(job.possible_backstories.len && H.client.prefs.backstory)
 		pick_backstory(job.possible_backstories, H)
+		H.set_squad_huds()
```
Comment (mattmatterson111) [inline code/controllers/subsystems/jobs.dm:479]: ?????

Comment (thedarkcrusader) [reply code/controllers/subsystems/jobs.dm:479]: Drunkeness  this is obviously not done and I have to relook at it again so ugh.

**Comment by mattmatterson111** [comment]
I'm doing this myself.

**Pull request closed without merging**